src/main/java/org/isf/visits/manager/VisitManager.java

```diff
@@ -88,10 +89,24 @@ public void validateVisit(Visit visit) throws OHServiceException {
 //							OHSeverityLevel.ERROR));
 //
 //		}
+		if (visit.getWard().getBeds() <= 0) {
+			errors.add(new OHExceptionMessage(MessageBundle.getMessage("angal.common.error.title"),
+					MessageBundle.getMessage("angal.visit.thewardhasnobedsavailable.msgg"),
+					OHSeverityLevel.ERROR));
+		}
```

Comment:
this validation is unnecesary: the visit is not related to the beds occupation, anyway the .getBeds() method only returns the number of beds in the wards (not its state) so it cannot be negative

Reply:
The test is for zero or less beds; not just negative.

So how do I test "to show also wards with no beds" or do we care if there are no beds in the ward?   The Jira issue seems to indicate that there is some test.

Reply:
No, in this case we don't care if there are available free beds (that if they are zero cannot be available in any case :) )

The validation is only about sex. The wards to be shown (in gui) are all of them, full list ("to remove list filtering")

I've also updated the issue description because it had a wrong link (OP-704 -> OP-714)

Reply:
removed the test

Reply:
Thanks you @dbmalkovsky !